src/templates/queues/base_queues.html

```diff
@@ -0,0 +1,14 @@
+{% extends "base.html" %}
+
+{% block extra_headers %}
+    <link rel="stylesheet" href="{{ STATIC_URL }}forums/forums.css">
```

Comment:
Should this be done with `{% load static %} ... {% static url ... %}` instead? Also, is forums.css the right one to load in?